ikenna-portfolio/portfolio/target/portfolio-1/index.html

```diff
@@ -22,6 +22,7 @@ <h1>Ikenna Elue's Portfolio</h1>
       <br>
       <p>Click <a href="calisthenics.html">here</a> to see some calisthenics moves I'm working towards!</p>
       <p>Click <a href="projects.html">here</a> to see/read some cool projects I'm working on!</p>
+      <p>Click <a href="test.html">here</a> to see test.</p>
```

Comment:
Consider rewording this to be more descriptive.

Reply:
done